src/extension.ts

```diff
@@ -64,7 +65,11 @@ export function activate(context: vscode.ExtensionContext) {
         }));
     context.subscriptions.push(vscode.commands.registerCommand(
         `${SETTINGS_VIEW_NAME}.refresh`,
-        () => cdpTargetsProvider.refresh()));
+        async () => {
+            cdpTargetsProvider.refresh();
+            await cdpTargetsProvider.getChildren();
+            vscode.window.showInformationMessage("Refresh complete");

```

Comment:
Have you thought about adding the notification to the end of `getChildren` so that it triggers on all refreshes?  Currently this only triggers if you press the refresh button and does nothing when destroying/creating new targets.

Reply:
I wasn't sure if we want to announce it when creating/destroying targets because it will produce a lot more notifications and might get cluttered (I haven't seen a way to hide notifications after an amount of time yet, lmk if you know about that). Plus, if targets are created/destroyed by opening/closing tabs in the browser, the screen reader will already announce that the tab is being open/closed.

Putting it at the end of `getChildren` would be good though because it's eventually called anyway after `cdpTargetsProvider.refresh()` (the call to `getChildren` here is redundant but helps with getting a more accurate timing for the announcement). Also, I think there might be a similar accessibility issue with the close target button, that this would solve.